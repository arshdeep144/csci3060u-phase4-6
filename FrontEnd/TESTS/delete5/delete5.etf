10 admin01 AD 99999
02 user09  0
00 admin01 AD 99999
