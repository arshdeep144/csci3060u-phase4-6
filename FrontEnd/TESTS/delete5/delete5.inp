login
admin01
deleteuser
user09
logout
