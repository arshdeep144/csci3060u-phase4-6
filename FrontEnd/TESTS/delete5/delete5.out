Welcome to the Online Auction System! Please login to Proceed
Please enter your username: 
Welcome to the main menu. Please enter a command to continue: 
Please enter a username
The record with the name user09 has been deleted if it exsisted
Welcome to the main menu. Please enter a command to continue: 
